app/enterprise/2.3.x/deployment/upgrades/migrations.md

```diff
@@ -15,65 +15,118 @@ to {{site.ee_product_name}}. See
-  * **Important:** If you are currently running in hybrid mode, upgrade the Control Plane first, and
-    then the Data Planes.
-  * If you are currently running 2.2.x in classic (traditional)
+* If you are adding a new plugin to your installation, you need to run
+  `kong migrations up` with the plugin name specified. For example,
+  `KONG_PLUGINS=oauth2`.
+
+### Hybrid mode considerations
+
+<div class="alert alert-ee blue">
+<strong>Important:</strong> If you are currently running in 
+[hybrid mode](/enterprise/{{page.kong_version}}/deployment/hybrid-mode/), 
+upgrade the Control Plane first, and then the Data Planes.
+</div>
+
+* If you are currently running 2.3.x in classic (traditional)
   mode and want to run in hybrid mode instead, follow the hybrid mode
   [installation instructions](/enterprise/{{page.kong_version}}/deployment/hybrid-mode-setup/)
   after running the migration.
-  * Custom plugins (either your own plugins or third-party plugins that are not shipped with Kong)
+* Custom plugins (either your own plugins or third-party plugins that are not shipped with Kong)
   need to be installed on both the Control Plane and the Data Planes in Hybrid mode. Install the
   plugins on the Control Plane first, and then the Data Planes.
-  * The [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced) plugin does not
+* The [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced) plugin does not
     support the `cluster` strategy in hybrid mode. The `redis` strategy must be used instead.
-* If you are adding a new plugin to your installation, you need to run
-  `kong migrations up` with the plugin name specified. For example,
-  `KONG_PLUGINS=oauth2`.
 
+### Kong for kubernetes considerations
+
+The Helm chart automates the upgrade migration process. When running `helm upgrade`, 
+the chart spawns an initial job to run `kong migrations up` and then spawns new 
+Kong pods with the updated version. Once these pods become ready, they begin processing 
+traffic and old pods are terminated. Once this is complete, the chart spawns another job 
+to run `kong migrations finish`.
+
+While the migrations themselves are automated, the chart does not automatically ensure 
+that you follow the recommended upgrade path. If you are upgrading from more than one minor 
+Kong version back, check the upgrade path recommendations for Kong open source or Kong Gateway.
+
+Although not required, users should upgrade their chart version and Kong version indepedently. 
+In the event of any issues, this will help clarify whether the issue stems from changes in 
+Kubernetes resources or changes in Kong.
+
+Users may encounter an error when upgrading which displays a large block of text ending with 
+`field is immutable`. This is typically due to a bug with the `init-migrations` job, which was 
+not removed automatically prior to 1.5.0. If you encounter this error, deleting any existing 
+`init-migrations` jobs will clear it.
```

Comment:
Should be okay to clear this out. 1.5 is ancient and new installs should not use it.
```suggestion
```